src/sentry/models/organizationmember.py

```diff
@@ -89,6 +97,18 @@ class OrganizationMember(Model):
     teams = models.ManyToManyField(
         "sentry.Team", blank=True, through="sentry.OrganizationMemberTeam"
     )
+    inviter = FlexibleForeignKey(
+        settings.AUTH_USER_MODEL, null=True, blank=True, related_name="sentry_inviter_set"
+    )
+    invite_status = models.PositiveSmallIntegerField(
+        choices=(
+            (InviteStatus.APPROVED.value, _("Approved")),
+            (InviteStatus.REQUESTED_TO_BE_INVITED.value, _("Org member requested to invite user")),
+            (InviteStatus.REQUESTED_TO_JOIN.value, _("User requested to join org")),
+        ),
+        default=InviteStatus.APPROVED.value,
+        null=True,
+    )
```

Comment:
I wonder if we should backfill this as well

Reply:
Should that be in the same migration or separate?

Reply:
Sounds like this should be separate. I'll followup with a PR to backfill

Reply:
👍